projectile.el

```diff
@@ -905,10 +905,10 @@ will return the current directory, otherwise it'd raise an error."
   ;; instead
   (or (cl-subst nil 'none
                 (or (and projectile-cached-buffer-file-name
-                         (equal projectile-cached-buffer-file-name buffer-file-name)
+                         (equal projectile-cached-buffer-file-name (or buffer-file-name 'none))
```

Comment:
Btw, we could have also wrapped this in a `(when buffer-file-name` which might have been more clear, but this came to my mind only after I merged this. Anyways, not a big deal.

Reply:
Wrapped which? I tried wrapping the whole thing in that previously and it broke some functionality for me.